### PR TITLE
Add membership manipulation to oo-admin-ctl-domain

### PR DESCRIPTION
**Pull request by tiwillia**
Bug 1145344
Bugzilla link https://bugzilla.redhat.com/show_bug.cgi?id=1145344
Adds the ability to manipulate and list membership of a domain through the oo-admin-ctl-domain tool.

### CODE REVIEW

**Comment by tiwillia** [comment]
[test]

File: broker-util/oo-admin-ctl-domain [hunk 1]
```diff
@@ -285,6 +293,174 @@ when "delete"
...
+  begin
+    owner = CloudUser.with(consistency: :eventual).find_by(login: login)
+  rescue Mongoid::Errors::DocumentNotFound
+    puts "User with login '#{login}' not found"
+    exit 1
+  end
+  begin
+    user = CloudUser.with(consistency: :eventual).find_by(login: member)
+  rescue Mongoid::Errors::DocumentNotFound
+    puts "User with login '#{login}' not found"
```
Comment (brenton) [inline broker-util/oo-admin-ctl-domain]: This should probably be 'member'.

**Comment by tiwillia** [comment]
https://github.com/openshift/origin-server/pull/6140

**Comment by tiwillia** [comment]
https://github.com/openshift/origin-server/pull/6139

**Comment by brenton** [comment]
Great work with this.  QE seems happy. :) Could you take a shot at adding at least a few happy path test cases to the cucumber suite?  See controller/test/cucumber/platform-oo-admin.feature.

**Comment by tiwillia** [comment]
[test]

**Comment by tiwillia** [comment]
[test] one more time

**Comment by brenton** [comment]
[merge] Thanks for the tests!

**Pull request merged**